src/Game.cpp

```diff
@@ -358,6 +359,10 @@ void Game::TimeStep(float step)
 
 	SfxManager::TimeStepAll(step, m_space->GetRootFrame());
 
+	// post these where they can be accessed without dependencies anywhere
```

Comment:
It would be nice to have some kind of globally visible blackboard where these could be posted each update and queried at will. What's the better way of doing this?